CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.5.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.5.0) - 2023-11-28
+
+This release avoids a panic when reading cached sprout or history trees in Zebra v1.4.0, 
+improves how Zebra recovers from network interruptions, avoids potential network hangs, and
+limits the ability of synthetic nodes to spread throughout the network through Zebra to
+address some of the Ziggurat red team report.
+
+Finally, we've added an experimental "shielded-scan" feature and the zebra-scan crate as steps
+towards supporting shielded scanning in Zebra.
```

Comment:
There are multiple known security risks/issues in this code, do we need to warn people about not using important private keys with it?

There is a security list in #7728 but it's probably incomplete.